digits/config/caffe_option.py

```diff
@@ -12,8 +12,12 @@
 from . import prompt
 from digits import device_query
 from digits.utils import parse_version
+from digits.utils.errors import UnsupportedPlatformError
 
 class CaffeOption(config_option.FrameworkOption):
+    version_string = None
+    flavor = None
```

Comment:
Let's not mix instance attributes and class attributes. `wtforms` does that and it's super confusing.